server/lobbyconnection.py

```diff
@@ -202,6 +203,18 @@ async def on_message_received(self, message):
             })
             if not e.recoverable:
                 await self.abort(e.message)
+        except GameJoinError as e:
+            self._logger.warning(
+                "GameJoinError[%s]: %s",
+                self.user_agent,
+                e.message,
+            )
+            await self.send({
+                "command": "game_join_failed",
+                "style": "error",
+                "text": e.message,
+                "uid": e.uid
+            })
```

Comment:
I think probably the error class isn't necessary. We can just send the messages directly

Reply:
you are correct. fixed